lib/express/websocket.js

```diff
@@ -0,0 +1,66 @@
+import _ from 'lodash';
+import url from 'url';
+
+
+const WEBSOCKET_SERVERS = {};
```

Comment:
why do we have multiple servers? is it one per session? one per driver? can we encode that in messages instead?

Reply:
it is a bit more tricky than it looks. Have you seen the example at https://github.com/websockets/ws/pull/885  ?

Reply:
I guess what I'm asking is, why do we need multiple websocket servers in the first place? Wouldn't just one do?

Reply:
not really. I am considering an example where we have multiple parallel sessions running on a single server. Then we'll need a separate websocket for each of them. Also, there might be different websockets for different endpoints, for example one websocket may broadcast logs and the other one - screenrecording data.

Reply:
OK. that sounds reasonable. However if there's any reason to have just one server, I still think we could do it (by encoding session id and message type into the message itself). But if there's not really much expense in having multiple servers, then this seems OK to me.